WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -291,7 +291,7 @@ import Gridicons
         featuredImageView.image = nil
         var url = featuredImageURL
         featuredImageDesiredWidth = featuredImageView.frame.width
-        let size = CGSize(width:featuredImageDesiredWidth, height:100)
+        let size = CGSize(width:featuredImageDesiredWidth, height:featuredImageView.frame.height)
```

Comment:
I'm not sure if the frame's height i reliable enough to use for fetching our desired layout's height. I think there are cases when this might be fired, prior to the final layout of the cell, resulting in a slightly incorrect frame height. Would be more reliable or guaranteed to derive the height from the constraint constant.